sources/web/datalab/static/appbar.html

```diff
@@ -12,12 +12,12 @@
     <span class="autosave_status"></span>
   </span>
   <div class="btn-toolbar pull-right right-toolbar">
-    <div class="btn-group">
-      <button id="ungitButton" title="Open ungit on the notebooks repository" class="toolbar-btn">
+    <div id="ungitButton" class="btn-group">
```

Comment:
I've seen people use divs as buttons, but I don't recall seeing a div used as a button, with a button element inside it. This  seems a bit confusing to me. Why do you need to move the click handler to the div? Perhaps that deserves a comment.